brownie/scripts/test_multisig_as_governor.py

```diff
@@ -0,0 +1,77 @@
+from world import *
+from brownie import chain
+
+def main():
+  with TemporaryFork():
```

Comment:
One more request on for a test here - let's test giving access to multisig role.

Reply:
Wdym? Like when not running the deployment?

Reply:
Sorry - I garbled some words:

I mean have the test to verify that we can use the multisig address to control the timelock to add roles to the timelock. This ensures that we can easily transition away from having the multisig the only control on the timelock.

Reply:
Have added it now